kytos/core/interface.py

```diff
@@ -155,8 +155,11 @@ def __init__(self, name, port_number, switch, address=None, state=None,
         self._tag_lock = Lock()
         self.available_tags = {'vlan': self.default_tag_values['vlan']}
         self.tag_ranges = {'vlan': self.default_tag_values['vlan']}
+        self.special_available_tags = {'vlan': self.default_special_tags}
+        self.special_tag_range = {'vlan': self.default_special_tags}
```

Comment:
Overall looks good, but `special_tag_range` naming might leave room for confusion since it's not an actual range and just a list (behaving like a set of values), should we rename this to just `special_tags`? Same equivalent discussion to https://github.com/kytos-ng/topology/pull/178#discussion_r1409315032

Reply:
Name changed